dashboard/api.py

```diff
@@ -492,7 +493,21 @@ def is_exam_schedulable(user, course):
     """
     Check if a course is ready to schedule an exam or not
     """
-    schedulable_exam_runs = ExamRun.get_currently_schedulable(course)
+    now = now_in_utc()
+    # select course.
+    course_filter = Q(course=course)
+
+    # select current and future exam runs.
+    current_exam_runs_filter = Q(
+        date_first_schedulable__lte=now,
+        date_last_schedulable__gte=now,
+        date_last_eligible__gte=now.date()
+    )
+    future_exam_runs_filter = Q(date_last_eligible__gte=now.date())
+
+    schedulable_exam_runs = ExamRun.objects.filter(
+        course_filter & (current_exam_runs_filter | future_exam_runs_filter)
```

Comment:
Given the truth table it is sufficient to just filter `ExamRun`s  by `date_last_eligible__gte=now.date()`. Where `date_first_schedulable` and `date_last_schedulable` do not effect the result.

Reply:
So you just need this `schedulable_exam_runs = ExamRun.objects.filter(course=course, date_last_eligible__gte=now.date())`